lib/facility.rb

```diff
@@ -1,14 +1,64 @@
 class Facility
-  attr_reader :name, :address, :phone, :services
-
-  def initialize(name, address, phone)
+  attr_reader :name, :address, :phone,
+              :services, :registered_vehicles, :collected_fees
+              :administer_written_test
+  def initialize(name:, address:, phone:)
     @name = name
     @address = address
     @phone = phone
     @services = []
+    @registered_vehicles = []
+    @collected_fees = 0
+    @administer_written_test
```

Comment:
We don't need this here since this isn't actually state we want to keep track of. This is a behavior and you have a method by the same name that executes that behavior. Additionally, we wouldn't need this in the attr_readers